RecruitingEducatorsDB/4 reports.sql

```diff
@@ -0,0 +1,35 @@
+--Reports:
+--1. I need to know how many students from each college we were able to find jobs for in under 2 weeks. 
+--    This will help my company know which college's graduates we should try to attract to our company.
+select e.CollegeAttended, AmntOfStudents = count(*)
+from Educators e 
+where datediff(day, DateContacted, DateFoundJob) < 14
+group by e.CollegeAttended
+
+--2. Who were we more successful in placing, males or females?
+select top 1 e.Gender
```

Comment:
To make it clearer what the results are, include the total. And you don't really need `top 1`.

Reply:
Fixed